src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -396,26 +401,29 @@ public void dumpInventoryIntoWareHouse(@NotNull final InventoryCitizen inventory
      * @return the tile entity of the chest
      */
     @Nullable
-    private TileEntityChest searchRightChestForStack(@NotNull final ItemStack stack)
+    private IItemHandler searchRightItemHandler(@NotNull final ItemStack stack)
     {
-        if(InventoryUtils.findFirstSlotInInventoryWith(this, stack.getItem(), stack.getItemDamage()) != -1 && InventoryUtils.getOpenSlot(this) != -1)
+        if(InventoryUtils.findFirstSlotInInventoryWith(this.getItemHandler(), stack.getItem(), stack.getItemDamage()) != -1 && InventoryUtils.getOpenSlot(this.getItemHandler()) != -1)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)